R/zzz.R

```diff
@@ -0,0 +1,17 @@
+# Stuff to set up on load, run this last (hence the filename)
+# Taken from googlesheets3/zzz.R
+.onLoad <- function(libname, pkgname) {
+
```

Comment:
My recollection of the googlesheets package is it has this onload function because it provides a oauth that any user can use (with a client & secret), so I’m not sure the utility if it’s just left to “blah” (but I could be forgetting/missing something!)

Reply:
@lucymcgowan You are 100% correct as it stands. But @ledell mentioned registering a dedicated OAuth consumer for the `meetupr` project, so I took out my key that i used to teat the defaults, and left it open for the dedicated key. I *think* I checked the "maintainers can commit" box, so you should be able to add it to the PR directly (or I can, of course).